test/SpokePoolClient.v3Events.ts

```diff
@@ -263,9 +236,7 @@ describe("SpokePoolClient: Event Filtering", function () {
       // destinationChainId is appended by the SpokePoolClient for V3FundsDeposited events, so verify its correctness.
       expect(fillEvent.destinationChainId).to.equal(destinationChainId);
 
-      const expectedOutputToken = isV2Fill(fillEvent)
-        ? expectedFill.args!.destinationToken
-        : expectedFill.args!.inputToken;
+      const expectedOutputToken = expectedFill.args!.inputToken;
```

Comment:
Yeah, tsc seems unhappy about it otherwise:
![warn](https://github.com/across-protocol/sdk-v2/assets/108695806/abd372fa-4792-4a6c-ab43-29b4d9a8350c)